orchagent/tunneldecaporch.cpp

```diff
@@ -40,19 +41,29 @@ void TunnelDecapOrch::doTask(Consumer& consumer)
         string key = kfvKey(t);
         string op = kfvOp(t);
 
-        IpAddresses ip_addresses;
-        IpAddress src_ip;
+        IpAddresses dst_ip_addresses;
+        IpAddress src_ip_address("0.0.0.0");
         IpAddress* p_src_ip = nullptr;
         string tunnel_type;
         string dscp_mode;
         string ecn_mode;
         string encap_ecn_mode;
         string ttl_mode;
+        sai_object_id_t dscp_to_dc_map_id = SAI_NULL_OBJECT_ID;
+        sai_object_id_t tc_to_pg_map_id = SAI_NULL_OBJECT_ID;
+        TunnelTermType term_type = TUNNEL_TERM_TYPE_P2MP;
```

Comment:
I don't think we need to have this variable assigned here. In the calling function, based on p_src_ip, this can be determined. Please avoid and extra variable being passed

Reply:
Fixed. The variable `term_type` is removed now. But `dscp_to_dc_map_id ` and `tc_to_pg_map_id` are required and can't be removed.